pkg/function/examples/event_time_filter/Makefile

```
@@ -0,0 +1,12 @@
+.PHONY: build
+build:
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -v -o ./dist/event-time-filter-example main.go
+
+.PHONY: image
+image: build
+	docker build -t "quay.io/numaio/numaflow-go/mapt-event-time-filter::latest" --target event-time-filter .
```

```suggestion
	docker build -t "quay.io/numaio/numaflow-go/mapt-event-time-filter:latest" --target event-time-filter .
```

I have never seen `::` syntax in image tag, is it valid?

---

oops. typo